tools/advisor/advisor/rules.ini

```diff
@@ -29,7 +29,6 @@ conditions=stall-too-many-memtables
 [Condition "stall-too-many-memtables"]
 source=LOG
```

Comment:
Where do we explain the format to the users? If we do not have such place already should we do it as inline comment on top of this file?